docs/design/kata-guest-image-management-design.md

```diff
@@ -113,6 +113,15 @@ Next, the kata-agent's RPC module will handle the create container request which
 > **Notes:**
 > In this flow, `ImageService.pull_image()` parses the image metadata, looking for either the `io.kubernetes.cri.container-type: sandbox` or `io.kubernetes.cri-o.ContainerType: sandbox` (CRI-IO case) annotation, then it never calls the `image-rs.pull_image()` because the pause image is expected to already be inside the guest's filesystem, so instead `ImageService.unpack_pause_image()` is called.
 
+## Using guest image pull with `nerdctl`
+
+The minimum required `nerdctl` version is `v2.0.0-rc0`, as that version has added the `--annotation` flag, which take advantage of.
+
+When running a workload, do:
```

Comment:
Optional suggestion to maybe make it clearer?
```suggestion
When running a workload, add the `--annotation io.kubernetes.cri.image-name=<image>` option e.g.:
```

Reply:
Done, thanks @stevenhorsman!